### DISCUSSION TITLE
also update cargo binary packages

### DISCUSSION THREAD

**Discussion by Simon-Laux** [Morganamilo/paru#576] [Ideas]
that were installed with `cargo install`.
paru uses rust, many rust-cli tools can be installed with cargo.

Since cargo packages are also source code even the reviewing of the source could be added.

I'm not sure if it's worth the effort though, also if updating is possible soon people want also be able to install from crates.io and so this feature idea could easily outgrow the scope of the project.

**Comment by Morganamilo** [comment]
This is way out of score. Just because paru happens to be written in rust is incidental to its goals.
